tests/test_transform.py

```diff
@@ -293,7 +293,7 @@ def test_xy_input(rows, cols, exp_xy, aff):
 
 
 @pytest.mark.parametrize("aff", [Affine.identity()])
-@pytest.mark.parametrize("rows, cols", [([0, 1, 2], [0, 1]), ("0", "0")])
```

Comment:
NumPy translates these numerical strings to floats, so this is no longer an error. Not good usage, but no longer an error.